src/docs/admin/ProActiveAdminGuide.adoc

```diff
@@ -188,435 +188,730 @@ If a problem occurs, check output in /var/log/proactive/scheduler
-        retries: 30
-
-  proactive-node:
-    image: dockerhub.activeeon.com/${RELEASE}/proactive-node:${PROACTIVE_VERSION}
-    hostname: service-node
-    container_name: proactive-node
-    networks:
-      - pa-network
-    env_file:
-      - .env
-    # enable DOCKER_HOST on Windows machine when you want to run docker-in-docker
-    #environment:
-    #    - DOCKER_HOST=tcp://host.docker.internal:2375
-    volumes:
-      - /tmp
-      - /var/run/docker.sock:/var/run/docker.sock:rw
-      - node-default:/opt/proactive/node/default
-      - node-previous:/opt/proactive/node/previous
-    depends_on:
-      proactive-server:
-        condition: service_healthy
 
-volumes:
-  proactive-default:
-  proactive-previous:
-  node-default:
-  node-previous:
+==== Configure the Docker Compose Input File
+
+Before starting the ProActive installation, you need first to configure the installation input parameters using the Docker Compose environment file `.env`.
 
-networks:
-  pa-network:
+'''
+.*[underline blue]#Click here to display the .env input file#*
+[example%collapsible]
+====
+link:./references/automatedInstallScripts/.env[.env, title="Click to download"^]:
+[source,yaml]
 ----
+include::./references/automatedInstallScripts/.env[]
+----
+====
+'''
 
+===== Required configuration
+The `.env` file referenced above can be used as it is for a minimal installation. However, it requires the definition of five mandatory parameter values:
 
+- `DOCKER_IMAGE_PROACTIVE_NAMESPACE`: Defines the ProActive Docker image namespace (or repository) in the Activeeon private registry.
+- `DOCKER_IMAGE_PROACTIVE_TAG`: Defines the ProActive release version.
+- `DOCKER_IMAGE_POSTGRES_NAMESPACE`: Defines the ProActive Postgres namespace.
+- `DOCKER_IMAGE_POSTGRES_TAG`: Defines the Postgres version to be used.
+- `PWS_HOST_ADDRESS`: Defines the internal hostname or IP address of the machine where ProActive will be installed.
 
-===== Set up ProActive using PostgreSQL Database
+[[_compose_advanced_parameters]]
+===== Advanced configuration
+For more advanced configuration of the ProActive server, please refer to the parameters listed below:
 
-This Docker Compose configuration `docker-compose-postgres.yml` can be used to run the ProActive server container with an external PostgreSQL Database container.
-It will start up to three containers, one for the server, one for the Nodes, and one for the Database.
+[[_compose_PWS_Server_Configuration_Properties]]
+*_PWS Server Configuration Properties_*
 
-[source]
+This section outlines the global configuration options available to configure the ProActive server.
+
+- `PWS_PROTOCOL`: The internal http protocol to use (http or https). By default, http.
+
+- `PWS_PORT`: The internal port used (e.g., 8080 for http, 8443 for https). By default, 8080.
+
+- `PWS_PAMR_PORT`: The internal port to use for PARM communication. By default, 33647.
+
+- `PWS_JOB_CLEANUP_DAYS`: Remove jobs history older than (in days). By default, 30.
+
```

Comment:
Housekeeping mechanism that triggers jobs history cleaning on a regular basis.

Reply:
I am not explaining the concept here. 
"Remove jobs history older than (in days)" is more clear to understand.

I will change it to this:
"Remove job history older than the specified number of days"